infra/main.bicep

```diff
@@ -315,7 +315,7 @@ module backend 'core/host/appservice.bicep' = {
       // Optional login and document level access control system
       AZURE_USE_AUTHENTICATION: useAuthentication
       AZURE_ENFORCE_ACCESS_CONTROL: enforceAccessControl
-      AZURE_ENABLE_GLOBAL_DOCUMENTS_ACCESS: enableGlobalDocuments
+      AZURE_ENABLE_GLOBAL_DOCUMENT_ACCESS: enableGlobalDocuments
```

Comment:
FYI @mattgotteiner

Reply:
Was about to PR this, good to see it's maintained! This one was driving me crazy! 😆 
Thanks team!